docs/4.2/admin-guide.md

```diff
@@ -858,6 +858,41 @@ turing        d52527f9-b260-41d0-bb5a-e23b0cfe0f8f     10.1.0.5:3022      distro
 dijkstra      c9s93fd9-3333-91d3-9999-c9s93fd98f43     10.1.0.6:3022      distro:debian
 ```
 
+### Removing Nodes from a Cluster
+
+To remove a node from a cluster, simply stop the node that you want to remove. 
+
+#### TTL time
```

Comment:
Technically TTL is time-to-live to "time-to-live time" isn't quite right.

I'd just change this to "Time to live"

Reply:
revised